lisp/mastodon-tl.el

```diff
@@ -260,26 +306,49 @@ TIME-STAMP is assumed to be in the past."
      'toot-id      id
      'toot-json    toot)))
 
-(defun mastodon-tl--set-face (string face render)
-  "Set the face of a string. If `render' is not nil
-also render the html"
-  (propertize
-   (with-temp-buffer
-     (insert string)
-     (when render
-       (let ((shr-use-fonts nil))
-         (shr-render-region (point-min) (point-max))))
-     (buffer-string))
-   'face face))
+(defun mastodon-tl--render-text (string)
+  "Returns a propertized text giving the rendering of the given HTML string."
+  (with-temp-buffer
+    (insert string)
+    (let ((shr-use-fonts t)
```

Comment:
Can we make `shr-use-fonts` a flag? I'd much rather not use the shr fonts.

Reply:
Ah yes.  That shouldn't be here - has nothing to do with tabbed browsing.  I'll revert and re-add it gated by a var in a later change.